mailet/standard/src/test/java/org/apache/james/transport/mailets/AmqpForwardAttributeTest.java

```diff
@@ -184,33 +183,27 @@ public void serviceShouldThrowWhenAttributeContentIsNotAMap() throws MessagingEx
     }
 
     @Test
-    public void serviceShouldLogWhenTimeoutException() throws Exception {
+    public void serviceShouldNotFailWhenTimeoutException() throws Exception {
         mailet.init(mailetConfig);
         Mail mail = mock(Mail.class);
         when(mail.getAttribute(MAIL_ATTRIBUTE)).thenReturn(ATTRIBUTE_CONTENT);
         ConnectionFactory connectionFactory = mock(ConnectionFactory.class);
-        TimeoutException expectedLoggedException = new TimeoutException();
-        when(connectionFactory.newConnection()).thenThrow(expectedLoggedException);
+        when(connectionFactory.newConnection()).thenThrow(new TimeoutException());
         mailet.setConnectionFactory(connectionFactory);
 
         mailet.service(mail);
-
-        verify(logger).error(any(String.class), eq(expectedLoggedException));
     }
 
     @Test
-    public void serviceShouldLogWhenIOException() throws Exception {
+    public void serviceShouldNotFailWhenIOException() throws Exception {
         mailet.init(mailetConfig);
         Mail mail = mock(Mail.class);
         when(mail.getAttribute(MAIL_ATTRIBUTE)).thenReturn(ATTRIBUTE_CONTENT);
         ConnectionFactory connectionFactory = mock(ConnectionFactory.class);
-        IOException expectedLoggedException = new IOException();
-        when(connectionFactory.newConnection()).thenThrow(expectedLoggedException);
+        when(connectionFactory.newConnection()).thenThrow(new IOException());
         mailet.setConnectionFactory(connectionFactory);
 
         mailet.service(mail);
-
-        verify(logger).error(any(String.class), eq(expectedLoggedException));
```

Comment:
does it also break this test: https://github.com/linagora/openpaas-mailets/blob/master/src/test/java/com/linagora/james/mailets/GuessClassificationMailetTest.java#L668 ? This means we are not able to test that we are really logging something when we need to?